resources/ts/Components/UitIdWidget.tsx

```diff
@@ -22,7 +22,7 @@ export const UitIdWidget = ({
   const widgetConfig = useMemo(
     () =>
       JSON.stringify({
-        $schema: "https://assets.uit.be/uitid-widget/config-schema.json",
+        $schema: `${import.meta.env.VITE_UITID_WIDGET_URL}config-schema.json`,
```

Comment:
It would make more sense for me if the url from .env was /uitid-widget without trailing forward slash